docs/developer-docs/defi/tokens/ledger/setup/icp_ledger_setup.mdx

```diff
@@ -111,7 +126,27 @@ Open the `dfx.json` file in your project's directory. Replace the existing conte
         "id": {
           "ic": "ryjl3-tyaaa-aaaaa-aaaba-cai"
         }
-      }
+      },
+      "init_arg": "(variant {
+        Init = record {
+          minting_account = \"$MINTER_ACCOUNT_ID\";
+          initial_values = vec {
+            record {
+              \"$DEFAULT_ACCOUNT_ID\";
+              record {
+                e8s = 10_000_000_000 : nat64;
+              };
+            };
+          };
+          send_whitelist = vec {};
+          transfer_fee = opt record {
+            e8s = 10_000 : nat64;
+          };
+          token_symbol = opt \"LICP\";
+          token_name = opt \"Local ICP\";
+        }
+      })
+    "
```

Comment:
i dont think this will work, you can not provide environment variables here, you have to hardcode this

Reply:
if we did the same for the icrc1 ledger i oversaw it and it has to be changed as well

Reply:
Fixed